### DISCUSSION TITLE
How to add "managed_schema" in manifest

### DISCUSSION THREAD

**Discussion by erwanjugand** [wxt-dev/wxt#685] [Q&A] [answered]
To retrieve “managed” storage, we must define in the manifest"managed_schema" property of the "storage" 
Source : https://developer.chrome.com/docs/extensions/reference/manifest/storage

But it does not seem possible to define it since `DefineConfig()`

**Comment by aklinker1** [comment, marked as answer]
Ignore the type errors, and it work

**Reply by erwanjugand** [reply]
That's right ! Will it be supported by `UserManifest` in the future or is this too special a case?

**Reply by aklinker1** [reply]
Managed storage is a chrome-only, non-standard API. I may be adding it soon, but it will be a part of a larger type change to add all of chrome's types
